homeassistant/components/idasen_desk/sensor.py

```diff
@@ -0,0 +1,99 @@
+"""Representation of Idasen Desk sensors."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfLength
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DeskData, IdasenDeskCoordinator
+from .const import DOMAIN
+
+
+@dataclass
+class IdasenDeskSensorDescriptionMixin:
+    """Required values for IdasenDesk sensors."""
+
+    value_fn: Callable[[IdasenDeskCoordinator], float | None]
+
+
+@dataclass
+class IdasenDeskSensorDescription(
+    SensorEntityDescription,
+    IdasenDeskSensorDescriptionMixin,
+):
+    """Class describing IdasenDesk sensor entities."""
+
+
+SENSORS = (
+    IdasenDeskSensorDescription(
+        key="height",
+        name="Height",
+        icon="mdi:arrow-up-down",
```

Comment:
It would be good to start adding `translation_key`

https://developers.home-assistant.io/blog/2022/12/01/entity_translations?_highlight=transla

Reply:
Withings also has a "Height" string. Should I add this to the common ones in homeassistant/strings.json?

Reply:
Or maybe to sensor/strings.json

Reply:
No